gradio/components/audio.py

```diff
@@ -86,6 +87,7 @@ def __init__(
             elem_id: An optional string that is assigned as the id of this component in the HTML DOM. Can be used for targeting CSS styles.
             elem_classes: An optional list of strings that are assigned as the classes of this component in the HTML DOM. Can be used for targeting CSS styles.
             format: The file format to save audio files. Either 'wav' or 'mp3'. wav files are lossless but will tend to be larger files. mp3 files tend to be smaller. Default is wav. Applies both when this component is used as an input (when `type` is "format") and when this component is used as an output.
+            shareable: If True, will allow user to share generation on Hugging Face Spaces Discussions. Can also provide a list of strings and Components that will be concatenated into the title post.
```

Comment:
This docstring should include the behavior when `shareable` is `None` (default). Also, it's not clear to me what the behavior is when `shareable` is set to `True` (what is the title in that case?). Also, how does this concatenation happen?

I feel like this parameter is doing too much. What do you think about separating it into `show_share_button` (which mirrors the `show_copy_button` that other components have) and a separate `share_post_title` parameter?

```suggestion
            shareable: If True, will allow user to share generation on Hugging Face Spaces Discussions. Can also provide a list of strings and Components that will be concatenated into the title post.
```